docs/core-concepts/images.md

```diff
@@ -109,110 +109,6 @@ Available Commands:
 
 The following subcommands are available:
 
-#### `build`
```

Comment:
We can keep the section and just link to the gadget-devel overview page.

Reply:
I added the link back.